modules/gdscript/gdscript_parser.h

```diff
@@ -33,6 +33,7 @@
 
 #include "gdscript_cache.h"
 #include "gdscript_tokenizer.h"
+#include "scene/main/node.h"
```

Comment:
```suggestion
```
Irrelevant include for the header, probably false positive for the internal `Node`, I'd suggest turning off any IDE feature adding includes as they're usually wrong